pkg/expression/contextstatic/exprctx.go

```diff
@@ -171,8 +163,9 @@ func NewStaticExprContext(opts ...StaticExprCtxOption) *StaticExprContext {
 		},
 	}
 
-	ctx.canUseCache = &atomic.Bool{}
-	ctx.canUseCache.Store(true)
+	cacheTracker := contextutil.NewPlanCacheTracker(ctx.evalCtx)
+	ctx.planCacheTracker = &cacheTracker
+	ctx.planCacheTracker.EnablePlanCache()
```

Comment:
How about to move this assign range apply opts:

```
if ctx. planCacheTracker == nil {
	cacheTracker := contextutil.NewPlanCacheTracker(ctx.evalCtx)
	ctx.planCacheTracker = &cacheTracker
	ctx.planCacheTracker.EnablePlanCache()
}
```

Reply:
Done.